### PR TITLE
Add bufr_version as global attribute to bufr2ioda output (for AMVs only)

### PR DESCRIPTION
**Pull request by ilianagenkova**
## Description
See Issue [#587](https://github.com/JCSDA-internal/ioda-converters/issues/587)

We are adding global attribute _bufr_verson_ (BUFR sequence version) to the ioda output from bufr2ioda, allowing us to distinguish between two BUFR versions used for AMVs (satellite winds).
Later, UFO filters could read  bufr_verson and make quality control decisions.

This PR adds code to:
ioda-bundle/iodaconv/tools/run_satwnds/satwnds_old_subset_template.yaml
ioda-bundle/iodaconv/tools/run_satwnds/satwnds_new_subset_template.yaml
to add the global attribute.

Link the issues to be closed with this PR
- fixes #<587>

## Acceptance Criteria (Definition of Done)
The ioda output will show bufr_version:

// global attributes:
		string :_ioda_layout = “ObsGroup” ;
		:_ioda_layout_version = 0 ;
		string :bufr_version@MetaData = “310077" ;  
 
(or "310014")

## Dependencies
None

## Impact
- [iodaconv] iodaconv in ioda-bundle

### CODE REVIEW

File: tools/run_satwnds/satwnds_new_subset_template.yaml
Comment (rmclaren) [inline tools/run_satwnds/satwnds_new_subset_template.yaml:78]: It might make sense to make the name something like seq_version and and the value equivalent to the mnemonic for that sequence (GEOSTWND or whatever), up to you :)

File: tools/run_satwnds/satwnds_old_subset_template.yaml
Comment (rmclaren) [inline tools/run_satwnds/satwnds_old_subset_template.yaml:73]: Same here.

File: tools/run_satwnds/satwnds_new_subset_template.yaml
Comment (ilianagenkova) [reply tools/run_satwnds/satwnds_new_subset_template.yaml:78]: bufr_version, seq_version, or even bufr_seq_version - I'm fine with either choice.

I used "310077"  b/c it comes straight from WMO documents. In fact, it should be "3-10-077". GEOSTWND is more familiar to NCEP folks, I agree, but I don't think MetOffice/NRL /etc are familiar with our naming in Table D.

Let's get @emilyhcliu to chime in, she is leading the Obs Re-engineering effort and may have an opinion on this too.

File: tools/run_satwnds/satwnds_new_subset_template.yaml
Comment (srherbener) [inline tools/run_satwnds/satwnds_new_subset_template.yaml:76]: There is no need to use the `@MetaData` suffix on an attribute name. The `@MetaData` syntax is for variable names, and the purpose of the suffix is to denote which group the variable belongs to. You might want to consider dropping the `@MetaData` suffix so it's a little easier to read the attribute name.

**Review by srherbener — APPROVED** [review]
I think the changes look good.

I put in a comment to consider, but I don't see that as a blocking issue for merging this PR into develop.

File: tools/run_satwnds/satwnds_new_subset_template.yaml
Comment (ilianagenkova) [reply tools/run_satwnds/satwnds_new_subset_template.yaml]: @srherbener,  I updated the code to remove @MetaData and renamed bufr_version with bufr_seq_version to accommodate @rmclaren's suggestion for clarity.
Also, I updated the string values, so they look exactly like in most WMO documents, thus easier to Google search for them. 

No more changes are anticipated for a while so merging would be fine. thanks!

Comment (srherbener) [reply tools/run_satwnds/satwnds_new_subset_template.yaml]: Thanks for making this change!

Comment (srherbener) [reply tools/run_satwnds/satwnds_new_subset_template.yaml]: Thanks for letting me know that you anticipate no more changes. I still need one more approval before merging.

**Review by rmclaren — APPROVED** [review]

**Pull request merged**
